3rdparty/jvm/org/pantsbuild/buck/util/zip/BUILD

```diff
@@ -0,0 +1,10 @@
+java_library(
```

Comment:
There is no corresponding change referencing this new target location leading to the CI errors. I hadn't noticed this on introduction, but we're getting away without declaring a source root here only because this library embed no resources. Maybe just punting until we hit the problem - if ever - is fine.